docs/runbooks/LowReadyVirtControllersCount.md

```diff
@@ -2,42 +2,45 @@
 
```

Comment:
Could you please add a comment indicating that this runbook has been edited? It makes automation a lot easier. Just something like `<!--edited by jherrman Nov 2022-->` is sufficient